README.md

```diff
@@ -1,79 +1,69 @@
 # It Is Always Soup
 
-The final team project for Makers.
-A tool to help a user find recipes from leftovers in the fridge.
+## Introduction
+The final team project for Makers Academy. A team of 3 were given just under 2 weeks to go from a vague brief to a full production application.
```

Comment:
Could we change this to 8 days to more accurately reflect the short amount of time?

Reply:
👍